### DISCUSSION TITLE
Possibilidade com Javascript

### DISCUSSION THREAD

**Discussion by douglasfab** [frontendbr/forum#1057] [Perguntas e Respostas]
É possível desenvolver um software (que pode ser um site e depois transformar em app) ou mesmo um script em uma página, que faça o seguinte:

1 - Consiga medir uma determinada área (o usuário clicar ou arrastar em um ponto inicial e um ponto final que ele quer medir, mas nada muito grande, no máximo seria até 2m o tamanho máximo do objeto).

2 - Depois disso, eu consigo filmar um vídeo e medir, por exemplo filmar um braço e começar a medir dos dedos até o ombro da pessoa, mas filmando ( ou pode ser uma foto panorâmica ou algo assim).

Isso possível? Se for possível você conhece quem saiba fazer isso? Não é um sistema, e sim um algoritmo que pode ser em uma tela apenas, desde que gere valor para ser armazenado em banco de dados.

**Comment by bernardodiasc** [comment]
É possível sim. Pelo celular usa-se a camera e o acelerômetro pra fazer a triangulação. Se voce quiser experimentar esse tipo de funcionalidade procura aplicativos de arquitetura. Alguns desses aplicativos ajudam a fazer medições de ambientes.
